src/index.d.ts

```diff
@@ -65,8 +65,13 @@ export interface MonacoEditorProps extends MonacoEditorBaseProps {
   /**
    * Refer to Monaco interface {monaco.editor.IEditorConstructionOptions}.
    */
-  options?: monacoEditor.editor.IEditorConstructionOptions;
+  optionsWithoutModel?: Omit<monacoEditor.editor.IEditorConstructionOptions, "model">;
```

Comment:
might be worth incorporating my other PR to this repo for this line: https://github.com/react-monaco-editor/react-monaco-editor/pull/288

the types are incorrect-ish